Shell-Assignment/wallpaper.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+path="."
+tmp=false
+output="pic.jpeg"
+resolution="1920/1080"
+url="https://picsum.photos/$resolution"
+
+print_usage() {
+    echo "Usage: $0 [ -p PATH | -t ] [ -o FILENAME ] [ -r RESOLUTION ]"
+    echo "    -p to save the image to a specific path."
+    echo "    -t to save the image in /tmp."
+    echo "    -o to specify the output filename."
+    echo "    -r to set a custom resolution for the image."
+    echo "Example: $0 -p ~/Desktop -o my-wallpaper -r 1300x720"
+    exit 1
+}
+
+while getopts ":p:tor:h" opt; do
+    case $opt in
+        p)
+            path="$OPTARG"
+            ;;
+        t)
+            tmp=true
+            ;;
+        o)
+            output="$OPTARG"
+            ;;
+        r)
+            resolution="$OPTARG"
+            url="https://picsum.photos/$resolution"
+            ;;
```

Comment:
As you have mentioned in your help section that the user will enter the resolution in "width x height" format and this resolution is placed in the url in "width / height" format. So, you should write the code to replace "x" with "/" in the string of resolution entered by the user.